scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -55,3 +55,21 @@ include::modules/installation-mirror-repository.adoc[leveloffset=+1]
 include::modules/ztp-acm-adding-images-to-mirror-registry.adoc[leveloffset=+1]
 
 include::modules/ztp-acm-installing-disconnected-rhacm.adoc[leveloffset=+1]
+
+// Creating custom resources
+
+include::modules/ztp-ztp-custom-resources.adoc[leveloffset=+1]
+
+include::modules/ztp-creating-siteconfig-custom-resources.adoc[leveloffset=+1]
+
+include::modules/ztp-discovery-iso.adoc[leveloffset=+2]
+
+include::modules/ztp-checking-the-managed-cluster-status.adoc[leveloffset=+2]
+
+include::modules/ztp-configuring-a-static-ip.adoc[leveloffset=+2]
```

Comment:
I think that moving this module up, possibly to line 65, would be better. (After reading your comments, moving the content into `ztp-creating-siteconfig-custom-resources.adoc` would be best, but it would be easier to find if you placed it closer to the place where the user's actually going to need it.)

Reply:
I can see your reasoning. I moved it.